Backend/lib/sessionizer-metrics/src/Lib/Types/Payload.hs

```diff
@@ -0,0 +1,58 @@
+{-
```

Comment:
Why to define payloads inside the lib?

Reply:
So that all the application have a common payload for the event. 
Instead of defining the payload again in all application.

Reply:
What do you mean by all application? And how the payload is common?

Reply:
Like We have to define it individually in RiderApp and DynamicOfferDriver